projects/plugins/jetpack/modules/masterbar/admin-menu/class-admin-menu.php

```diff
@@ -144,7 +144,8 @@ public function add_upgrades_menu( $plan = null ) {
 				$site_upgrades = sprintf(
 					$site_upgrades,
 					__( 'Upgrades', 'jetpack' ),
-					$plan
+					// phpcs:ignore WordPress.WP.I18n.NonSingularStringLiteralText
+					__( $plan, 'jetpack' )
```

Comment:
I'm surprised this works, as the phpcs error mentioned, having variables inside translatable strings doesn't typically work. :)

Reply:
The reason why it doesn't usually work and should be avoided is that we extract the translatable strings from the raw strings passed to the translation functions and with a variable there nothing would be extracted and translated.
In this case it works because the plan names are already mentioned in other places and translations exist for them, but this is an exception.